plugins/sfRadPlugin/lib/sfRadPlugin.class.php

```diff
@@ -105,7 +105,7 @@ public function setProperty($name, $value)
 
             case 'languageNotes':
                 $note = $this->resource->getMemoryNotesByType(['noteTypeId' => QubitTerm::LANGUAGE_NOTE_ID])->offsetGet(0);
-                $missingNote = 0 === count($note);
+                $missingNote = 0 === strlen($note);
```

Comment:
I think `$missingNote = empty($note);` would achieve the same result and be easier to read.